move/move.lua

```diff
@@ -24,9 +24,16 @@ function jumpdrive.move(source_pos1, source_pos2, target_pos1, target_pos2)
 	local t0 = minetest.get_us_time()
 
 	-- load areas (just a precaution)
-	if minetest.load_area then
-		minetest.load_area(source_pos1, source_pos2)
-		minetest.load_area(target_pos1, target_pos2)
+	if (tostring(minetest.settings:get_bool(jumpdrive.allow_jump_to_unloaded_area)) or "true") == "true" then
+		if minetest.emerge_area and minetest.load_area then
```

Comment:
Probably don't need to check for these, `load_area` has existed since 5.0, and `emerge_area` even earlier.

Reply:
What's the minimum version for the mod?

Reply:
Not sure exactly, but it's definitely at least 5.0.